lib/livebook/hubs/provider.ex

```diff
@@ -3,8 +3,8 @@ defprotocol Livebook.Hubs.Provider do
 
   alias Livebook.Secrets.Secret
 
-  @type t :: Livebook.Hubs.Enterprise.t() | Livebook.Hubs.Fly.t() | Livebook.Hubs.Local.t()
-  @type capability :: :connect | :secrets
+  @type t :: Livebook.Hubs.Enterprise.t() | Livebook.Hubs.Fly.t() | Livebook.Hubs.Personal.t()
+  @type capability :: :connect | :secrets | :list_secret | :create_secret | :delete_secret
```

Comment:
```suggestion
  @type capability :: :connect | :secrets | :list_secrets | :create_secret | :delete_secret
```

btw, what is the difference between :secrets and :list*, :creat* and :delete*?

Reply:
Since I'm not removing the current implementation, the `:secrets` remains until next PR, where I'll remove everything related to `App secrets` and start using it directly from the Personal Hub.